tikv-configuration-file.md

```diff
@@ -1432,6 +1432,10 @@ Configuration items related to Titan.
 
 ### `enabled`
 
+> **Warning**
+>
+> When disabling Titan for TiDB versions earlier than v8.5.0, it is not recommended to modify this configuration item to `false` to avoid TiKV crashes. To disable Titan, refer to the steps in [Disable Titan](/storage-engine/titan-configuration.md#disable-titan).
```

Comment:
```suggestion
> When disabling Titan for TiDB versions earlier than v8.5.0, it is not recommended to modify this configuration item to `false`, as this might cause TiKV to crash.. To disable Titan, refer to the steps in [Disable Titan](/storage-engine/titan-configuration.md#disable-titan).
```